wallet-new/src/Cardano/Wallet/API/WIP/Handlers.hs

```diff
@@ -0,0 +1,119 @@
+module Cardano.Wallet.API.WIP.Handlers where
+
+import           Universum
+
+import           Servant
+
+import           Cardano.Wallet.API.Request
+import           Cardano.Wallet.API.Response
+import           Cardano.Wallet.API.V1.Types as V1
+import qualified Cardano.Wallet.API.WIP as WIP (API)
+import           Cardano.Wallet.WalletLayer (PassiveWalletLayer)
+import qualified Cardano.Wallet.WalletLayer as WalletLayer
+
+import           Pos.Core.Common (Coin (..))
+
+-- | WIP @Servant@ handlers the are not part of the offical api yet.
+handlers :: PassiveWalletLayer IO -> ServerT WIP.API Handler
+handlers pwl = checkExternalWallet pwl
+           :<|> newExternalWallet pwl
+           :<|> deleteExternalWallet pwl
+
+-- | Creates a new or restores an existing @wallet@ given a 'NewWallet' payload.
+-- Returns to the client the representation of the created or restored
+-- wallet in the 'Wallet' type.
+newWallet :: PassiveWalletLayer IO
```

Comment:
Hm... I see our unimplemented endpoints in `wallet-new/src/Cardano/Wallet/API/WIP.hs`, and handlers for them here (`checkExternalWallet`, `newExternalWallet` and `deleteExternalWallet`), but why did you put `newWallet` handler here? Is it unimplemented handler too??

Reply:
ups
will fix it

Reply:
Thanks for spotting..I made a forced-update to fix this.